ladybug/datatype/distance.py

```diff
@@ -2,60 +2,60 @@
 """Distance data type."""
 from __future__ import division
 
-from ._base import DataTypeBase
+from .base import DataTypeBase
 
 
 class Distance(DataTypeBase):
     """Distance"""
-    name = 'Distance'
-    units = ['m', 'ft', 'mm', 'in', 'km', 'mi', 'cm']
-    min = 0
-    abbreviation = 'D'
+    _units = ('m', 'ft', 'mm', 'in', 'km', 'mi', 'cm')
+    _si_units = ('m', 'mm', 'km', 'cm')
+    _ip_units = ('ft', 'in', 'mi')
+    _min = 0
+    _abbreviation = 'D'
 
     def _m_to_ft(self, value):
         return value * 3.28084
 
     def _m_to_mm(self, value):
-        return value * 1000
+        return value * 1000.
 
     def _m_to_in(self, value):
         return value * 39.3701
 
     def _m_to_km(self, value):
-        return value / 1000
+        return value / 1000.
 
     def _m_to_mi(self, value):
         return value / 1609.344
 
     def _m_to_cm(self, value):
-        return value * 100
+        return value * 100.
 
     def _ft_to_m(self, value):
         return value / 3.28084
 
     def _mm_to_m(self, value):
-        return value / 1000
+        return value / 1000.
 
     def _in_to_m(self, value):
         return value / 39.3701
 
     def _km_to_m(self, value):
-        return value * 1000
+        return value * 1000.
 
     def _mi_to_m(self, value):
         return value * 1609.344
 
     def _cm_to_m(self, value):
-        return value / 100
+        return value / 100.
 
     def to_unit(self, values, unit, from_unit):
         """Return values in a given unit given the input from_unit."""
         return self._to_unit_base('m', values, unit, from_unit)
 
     def to_ip(self, values, from_unit):
         """Return values in IP given the input from_unit."""
```

Comment:
docstring suggests that the return is only values but the method returns both the values and the new unit. This is the case for almost all the conversion methods. I think we should clarify that in docstring to avoid confusion.

Reply:
Done